couchdb/tests/Makefile

```diff
@@ -1,15 +1,29 @@
 BACKSLASH := \$ #
-COUCHDB_PASSWORD := test_~!@\#$$%^&*_-+=`|$(BACKSLASH)(){}[]:;"'<>,.?/_pw
+COUCHDB_PASSWORD := pass
 COUCHDB_USER := medic-test-admin
 export COUCHDB_PASSWORD 
 export COUCHDB_USER
 
-.phony: test
-test:
-	docker compose up --build --wait
-	docker compose run --rm sut
+.PHONY: test wait-for-couchdb
+
+wait-for-couchdb:
+	@echo "Waiting for CouchDB containers to be ready..."
+	@for i in 1 2 3; do \
+		until curl -s -f -u "$(COUCHDB_USER):$(COUCHDB_PASSWORD)" http://localhost:$${i}5984/_up > /dev/null; do \
+			echo "Waiting for couchdb$$i..."; \
+			sleep 2; \
+		done; \
+	done
+	@echo "All CouchDB containers are ready"
+
+test: 
+	docker compose down -v --remove-orphans
+	docker compose up -d couchdb1 couchdb2 couchdb3
+	$(MAKE) wait-for-couchdb
+	npm install --no-save
```

Comment:
npm install should not be run. all dependencies should have already been installed in through the root env.